controllers/databasecluster_controller.go

```diff
@@ -129,13 +121,20 @@ wsrep_provider_options="gcache.size=%s"
 	finalizerDeletePGSSL            = "percona.com/delete-ssl"
 )
 
-var operatorDeployment = map[everestv1alpha1.EngineType]string{
-	everestv1alpha1.DatabaseEnginePXC:        pxcDeploymentName,
-	everestv1alpha1.DatabaseEnginePSMDB:      psmdbDeploymentName,
-	everestv1alpha1.DatabaseEnginePostgresql: pgDeploymentName,
-}
+var (
+	operatorDeployment = map[everestv1alpha1.EngineType]string{
+		everestv1alpha1.DatabaseEnginePXC:        pxcDeploymentName,
+		everestv1alpha1.DatabaseEnginePSMDB:      psmdbDeploymentName,
+		everestv1alpha1.DatabaseEnginePostgresql: pgDeploymentName,
+	}
 
-var maxUnavailable = intstr.FromInt(1)
+	maxUnavailable       = intstr.FromInt(1)
+	exposeAnnotationsMap = map[ClusterType]map[string]string{
+		ClusterTypeEKS: {
+			"service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout": "4000",
```

Comment:
I remember we removed the other LB annotations when we moved from PMM's DBaaS to everest, is this annotation required? Does it work without the other annotations we had previously?

Reply:
It does work without annotations